src/Orleans.sln

```diff
@@ -141,6 +141,19 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrleansZooKeeperUtils", "Or
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrleansCounterControl", "OrleansCounterControl\OrleansCounterControl.csproj", "{606B9647-CC0C-4058-BFBC-B5B7ED4F3C56}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Remote Deployment Scripts", "Remote Deployment Scripts", "{2EC6D21F-7590-4C57-8AF8-415A308D25D1}"
```

Comment:
Do we have to include those scripts in the solution? If yes, then they should be inside the solution folder. I think Sergey did not like it at some point.